### PR TITLE
chore(ui): replace run and save buttons with icons (#449)

### PR DESCRIPTION
**Pull request by rzhekova**
This PR will resolve issue #449. 

What I did as part of this PR was:
1. removed Cancel button as per issue
2. created a new component called IconButton - it's not actually a button, but behaves like one i.e. it's clickable, it can be disabled etc.
3. since now the buttons don't contain any text, I gave the div within the new component the title attribute and basically moved the text that was previously showing inside the old buttons in the title. So, now when a user hovers over the icons they will get a bit more context around what the icon buttons are for (when you hover over, a little message appears that says Schedule/Schedule & Run/Run
4. also added some CSS - the colour of the icons before and after hover are darker than the previous buttons but decided to keep them in the blue range

Let me know what you think.

image

### CODE REVIEW

**Comment by enudler** [comment]
hi @rzhekova 
thanks for this PR
really great work and much appreciated!

few comments that are due to our bad UX skills 

1. can we make the icons smaller (by 30 - 50%) so they can fit better to the form?
2. is it possible to change the save icon to the scheduler icon: https://fontawesome.com/icons/clock?style=regular
3. is there a play icon that is free and hollow (more white than blue/purple, maybe even less bold)?
4. is it possible to align the play icon to where the original run icon was (same line with the purple frame)
5. is it possible change to the color of the icons to the same color we use in the form; (#557eff)

**Comment by rzhekova** [comment]
Hey @enudler, yeah I can do all those things and you should have the changes at some point today. Just on point 3, there is another play icon that I can use https://fontawesome.com/icons/play-circle?style=regular and when I reduce the size it'll naturally become a bit thinner, but I won't be able to to use the even thinner (or lighter) version as that's only available to Pro users of fontawesome.

**Comment by enudler** [comment]
hey @rzhekova 
that's great :)
https://fontawesome.com/icons/play-circle?style=regular is good :)
I would go for 2x instead of 4x size

**Comment by rzhekova** [comment]
Hi @enudler just pushed the latest changes

File: ui/src/features/components/JobForm/index.js [hunk 1]
```diff
@@ -368,12 +370,27 @@ class Form extends React.Component {
...
+                <IconButton
+                  spinner={processingAction}
+                  disabled={!!this.isThereErrorOnForm()}
+                  onClick={() => this.whenSubmit(true)}
+                  inverted
+                  width='28px'
+                  height='28px'
+                  title={this.state.cron_expression ? 'Schedule & Run' : 'Run'}>
+                  <FontAwesomeIcon icon={faPlayCircle} size='2x' />
+                </IconButton>
```
Comment (manorlh) [inline ui/src/features/components/JobForm/index.js:393]: good job on the Ui side! 
thank you!

**Comment by enudler** [comment]
great work @rzhekova 
thank you very much
I am going to open another issue for the create a test form, will mention you if you want to take it.

**Pull request merged**